mail/mail/doctype/mail_account/mail_account.py

```diff
@@ -164,6 +165,15 @@ def validate_display_name(self) -> None:
 		if self.is_new() and not self.display_name:
 			self.display_name = frappe.db.get_value("User", self.user, "full_name")
 
+	def validate_backup_email(self) -> None:
+		"""Validates the backup email."""
+
+		validate_email_address(self.backup_email, True)
+
+		if not self.is_new():
```

Comment:
Validate backup_email should not be the same as the MailAccount.email. Except for the DMARC account.

Similar validation on Mail Account Request (email != account).